x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/rules/logical/OptimizerRules.java

```diff
@@ -7,9 +7,14 @@
 package org.elasticsearch.xpack.esql.optimizer.rules.logical;
```

Comment:
Isn't there a physical plan equivalent?

Reply:
Yes, `org.elasticsearch.xpack.esql.optimizer.PhysicalOptimizerRules`, however it does not seem to have an rules for `expressions`.